src/test/java/io/jenkins/plugins/fontawesome/api/FontAwesomeIconsTest.java

```diff
@@ -0,0 +1,69 @@
+package io.jenkins.plugins.fontawesome.api;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.apache.commons.lang.StringUtils;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EnumSource;
+import org.junit.jupiter.params.provider.NullSource;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * Test Class for {@link FontAwesomeIcons}.
+ *
+ * @author strangelookingnerd
+ */
+class FontAwesomeIconsTest {
+
+    @BeforeAll
+    static void init() {
+        assertTrue(
+                Files.exists(Paths.get("./target/classes/images/symbols/")),
+                "No Font Awesome Symbols found! Try running 'mvn process-resources' first!"
+        );
+    }
+
+    @Test
+    void testGetIconClassName() {
+        assertEquals("symbol-test plugin-font-awesome-api", FontAwesomeIcons.getIconClassName("test"));
```

Comment:
Seems that I forget to enable the architecture tests: I do not use JUnit assertions in my projects. Please rebase.